core/src/main/scala/org/apache/spark/sql/internal/SnappySessionState.scala

```diff
@@ -423,7 +423,8 @@ class SnappySessionState(snappySession: SnappySession)
 
     def apply(plan: LogicalPlan): LogicalPlan = {
       plan match {
-        case _: BypassRowLevelSecurity => plan
+        case  (_: BypassRowLevelSecurity | _: Update | _: Delete | _: DeleteFromTable) => plan
+
```

Comment:
Do we also need to add PutIntoTable?

Reply:
Thanks Shirish. You are right we would need PutInto to also be bypassed.
I will add it.